Dockerfiles/agent/Dockerfile

```
@@ -220,6 +220,11 @@ RUN find /etc -type d,f -perm -o+w -print0 | xargs -r -0 chmod g-w,o-w
 # Add Debian snapshot date for debugging
 RUN date +%Y%m%dT000000Z > .debian_repo_snapshot_date
 
+# Enable FIPS if needed
+RUN if [ -f /opt/datadog-agent/embedded/bin/fipsinstall.sh ]; then \
+      /opt/datadog-agent/embedded/bin/fipsinstall.sh; \
+    fi
```

This can fail if the `fipsinstall.sh` file exists but isn’t executable.

It would be more robust either to check that the script is executable:
```suggestion
RUN if [[ -x /opt/datadog-agent/embedded/bin/fipsinstall.sh ]]; then \
      /opt/datadog-agent/embedded/bin/fipsinstall.sh; \
    fi
```

or to force the interpreter:
```suggestion
RUN if [[ -f /opt/datadog-agent/embedded/bin/fipsinstall.sh ]]; then \
      /bin/bash /opt/datadog-agent/embedded/bin/fipsinstall.sh; \
    fi
```